new-lamassu-admin/src/components/SearchBox.js

```diff
@@ -0,0 +1,91 @@
+import InputBase from '@material-ui/core/InputBase'
+import Paper from '@material-ui/core/Paper'
+import { makeStyles } from '@material-ui/core/styles'
+import MAutocomplete from '@material-ui/lab/Autocomplete'
+import classnames from 'classnames'
+import React, { memo, useState, useEffect } from 'react'
+
+import { P } from 'src/components/typography'
+import { ReactComponent as SearchIcon } from 'src/styling/icons/circle buttons/search/zodiac.svg'
+
+import styles from './SearchBox.styles'
+
+const useStyles = makeStyles(styles)
+
+const SearchBox = memo(
+  ({
+    loading = false,
+    filters = [],
+    options = [],
+    inputPlaceholder = '',
+    size,
+    onChange,
+    ...props
+  }) => {
+    const classes = useStyles({ size })
+
+    const [popupOpen, setPopupOpen] = useState(false)
+
+    const inputClasses = {
+      [classes.input]: true,
+      [classes.inputWithPopup]: popupOpen
+    }
+
+    const innerOnChange = filters => onChange(filters)
+
+    // eslint-disable-next-line
+    useEffect(() => innerOnChange(filters), [filters])
```

Comment:
do we really need this useeffect here? seems like it's not doing anything. you already call innerOnChange outside useEffect